mysys_ssl/my_aes.cc

```diff
@@ -57,3 +57,99 @@ void my_aes_create_key(const unsigned char *key, uint key_length,
     *ptr^= *sptr;
   }
 }
+
+int my_aes_encrypt_nopad(const unsigned char *source, uint32 source_length,
+                         unsigned char *dest,
+                         const unsigned char *key, uint32 key_length,
+                         enum my_aes_opmode mode, const unsigned char *iv)
+{
+  unsigned char eiv[MY_AES_BLOCK_SIZE];
+  uint32 aligned_src_len= (source_length / MY_AES_BLOCK_SIZE)
+                          * MY_AES_BLOCK_SIZE;
+  enum my_aes_opmode ecb_mode;
+  int dst_len;
+
+  if (key_length == 64)
```

Comment:
This seems to be wrong. In AES 128 and 256 mean key sizes. Thus key length of 64 bytes means aes 512 and key length of 32 bytes means aes 256.